hack/build-cross.sh

```diff
@@ -33,6 +33,12 @@ os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
 OS_BUILD_PLATFORMS=("${OS_IMAGE_COMPILE_PLATFORMS[@]-}")
 os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
 
+# Build dockerregistry
+(
```

Comment:
It's only for incompatible package dependencies.  I'd prefer for this PR to
continue using godeps for the bulk of the changes (vendor doesn't solve
some of the challenges)

On Jun 9, 2016, at 10:39 AM, Michal Fojtik <notifications@github.com> wrote:

In hack/build-cross.sh
<https://github.com/openshift/origin/pull/8938#discussion_r66453366>:

> @@ -33,6 +33,12 @@ os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
>  OS_BUILD_PLATFORMS=("${OS_IMAGE_COMPILE_PLATFORMS[@]-}")
>  os::build::build_static_binaries "${OS_IMAGE_COMPILE_TARGETS[@]-}" "${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]-}"
>
> +# Build dockerregistry
> +(

@smarterclayton <https://github.com/smarterclayton> do you want switch this
PR to use vendor/ ?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/8938/files/bba3a7d7bd09c671c94531a774860945ca9e5953..4390daddfbe7ffb13344f4ef78362ff54aeb046a#r66453366>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p6bdItbHhELxxyK-hXieJ6GbSQunks5qKCWogaJpZM4IiLYZ>
.